xtream.patch

```diff
@@ -0,0 +1,59 @@
+Index: catalog/spatial/wfs/spatial-wfs-featuretransformer-xstream/src/main/java/org/codice/ddf/spatial/ogc/wfs/transformer/xstream/XStreamWfsFeatureTransformer.java
```

Comment:
❓ what is this?

Reply:
Left-over file from code-sharing between team members.  Removed.